python/tvm/relay/backend/vm.py

```diff
@@ -82,6 +82,10 @@ def _eval_vm(mod, ctx, *args):
         main_func = ir_pass.eta_expand(main_func.body, mod)
 
     assert isinstance(main_func, Function)
+    # optimize all functions in the module
+    for gvar, _ in mod.functions.items():
+        func_opt = optimize(mod[gvar], mod)
+        mod[gvar] = func_opt
     main_func = optimize(mod[mod.entry_func], mod)
```

Comment:
Is this unnecessary since you've optimized all functions in the mod?

Reply:
yes, i'll remove it.